### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by jayr0d** [eprbell/rp2#109] [General]
Filed my taxes.  No more testing of multiple platforms to figure out which works best.  No more sending data to yet another third-party.  No more deciding to use a platform only to have it not be available the following year.  No transaction limits.

Certainly not as newbie friendly as the web-based platforms, but once you know what you are doing DaLI and RP2 is certainly the way to go.

You should take donations.

Thanks again.

**Comment by eprbell** [comment]
Thanks for the nice words!

**Comment by haavardf** [comment]
I have been looking around for some solution to this. I have been debating if the work needed to get this set up was worth it or not. Jayr0d's comment made me deside to give it a go. :-) The biggest thing I have been looking for in this project is a simple way to get the data from an ethereum or bitcoin address. Looks like a great project. Wish me luck.

**Reply by jayr0d** [reply]
All of my imports are via CSV with DaLI.  You need to get an export from a data source and then do a conversion into the DaLI/RP2 format.

For ETH, you can export from etherscan.io.
For BTC, you can export from Bitcoin Core, or I am sure some web-based blockchain explorer if you are not using it.

Since I only do this once a year I always forget what to do so I do have steps written down if you get stuck.

**Comment by haavardf** [comment]
I got stuck just running the example yesterday, and made a ticker for it. I am sure it's a small thing, but it would prevent people from trying further. Both ETH and BTC import sounds find. Your write up would be greate to get. I am sure there are some golden nuggets in there. Thank you for your email.



Kind regards Haavard






---- On Sat, 27 Jul 2024 19:43:31 +1000 Jared Silva ***@***.***> wrote ---





All of my imports are via CSV with DaLI.  You need to get an export from a data source and then do a conversion into the DaLI/RP2 format.

For ETH, you can export from etherscan.io.
 For BTC, you can export from Bitcoin Core, or I am sure some web-based blockchain explorer if you are not using it.
Since I only do this once a year I always forget what to do so I do have steps written down if you get stuck.

**Reply by jayr0d** [reply]
Let's do one at a time, do you want BTC instructions first or ETH?  I would suggest starting with whatever crypto you bought first.  For example, if you used BTC to buy ETH, you will want to start with BTC.  Of course, this also assumes your BTC was on chain and not in an exchange.

**Reply by haavardf** [reply]
This has nothing to do with me, I was just trying to run the examples provided. They should run without a confusing exception.

**Reply by eprbell** [reply]
The exception problem is explained and solved in #126.

**Reply by jayr0d** [reply]
> This has nothing to do with me, I was just trying to run the examples provided. They should run without a confusing exception.

I was asking if you want my notes on how I import for BTC or ETH.

**Reply by haavardf** [reply]
Sorry, missed this question but I belive I said yes to that somewhere else. But I got it all up and running now, so all good. Thank you anyway.
